docs/docsite/rst/userguide/ee_reference.rst

```diff
@@ -2,304 +2,8 @@
-
-  **package_pip**
-    Package to install via pip for entrypoint support. This package will be installed in the final build image. The default value is ``dumb-init==1.2.5``.
-
-**package_manager_path**
-  A string with the path to the package manager (dnf or microdnf) to use. The default is ``/usr/bin/dnf``. This value will be used to install a Python interpreter, if specified in ``dependencies``, and during the build phase by the ``assemble`` script.
-
-**skip_ansible_check**
-  This boolean value controls whether or not the check for an installation of Ansible and Ansible Runner is performed on the final image. Set this value to ``True`` to not perform this check. The default is ``False``.
-
-**relax_passwd_permissions**
-  This boolean value controls whether the ``root`` group (GID 0) is explicitly granted write permission to ``/etc/passwd`` in the final container image. The default entrypoint script may attempt to update ``/etc/passwd`` under some container runtimes with dynamically created users to ensure a fully-functional POSIX user environment and home directory. Disabling this capability can cause failures of software features that require users to be listed in ``/etc/passwd`` with a valid and writeable home directory (eg, ``async`` in ansible-core, and the ``~username`` shell expansion). The default is ``True``.
-
-**workdir**
-  Default current working directory for new processes started under the final container image. Some container runtimes also use this value as ``HOME`` for dynamically-created users in the ``root`` (GID 0) group. When this value is specified, the directory will be created (if it doesn't already exist), set to ``root`` group ownership, and ``rwx`` group permissions recursively applied to it. The default value is ``/runner``.
-
-**user**
-  This sets the username or UID to use as the default user for the final container image. The default value is ``1000``.
-
-Example options section:
-::
-
-  options:
-      container_init:
-          package_pip: dumb-init>=1.2.5
-          entrypoint: '["dumb-init"]'
-          cmd: '["csh"]'
-      package_manager_path: /usr/bin/microdnf
-      relax_password_permissions: false
-      skip_ansible_check: true
-      workdir: /myworkdir
-      user: bob
-
-
-version
-~~~~~~~~
-
-An integer value that sets the schema version of the execution environment definition file. Defaults to ``1``. Must be ``3`` if you are using Ansible Builder 3.x.
-
+For detailed information about the |ee| definition,
```

Comment:
Does `|ee|` need to be defined in this file, or is it global?

Reply:
@gundalow i don't see it defined in this file, so i think it's global